src/styles/main.scss

```diff
@@ -1,3 +1,12 @@
-body {
-  margin: 0;
+@import '/src/styles/utils/varibeles';
+@import '/src/styles/blocks/header';
+@import '/src/styles/blocks/card';
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 1024px;
```

Comment:
```suggestion
  width: 1024px;
```

Reply:
should I focus on the tests first, or on the task, ignoring the tests? Because tests don't support grids, don't work with gaps...?